packages/zosfiles/src/api/methods/copy/Copy.ts

```diff
@@ -60,13 +58,10 @@ export class Copy {
             "request": "copy",
             "from-dataset": {
                 dsn: fromDataSetName,
-            }
+                member: fromMemberName,
+            },
         };
```

Comment:
We should enhance the `ICopyDataSetOptions` interface in order to properly type the payload : )
No need to do this right away.